packages/cmf-react-router/README.md

```diff
@@ -0,0 +1,36 @@
+# @talend/cmf-react-router
+
+This module is like cmf-router but for React router v5 (and more recent) where @talend/cmf-router is tied up with react-router v3.
```

Comment:
I think you should write a real doc and not rely on a package that will be removed when everyone migrates to this one